src/SFML/Window/DRM/ClipboardImpl.hpp

```diff
@@ -24,11 +24,10 @@
 
 #pragma once
 
-////////////////////////////////////////////////////////////
-// Headers
-////////////////////////////////////////////////////////////
-#include <SFML/System/String.hpp>
-
+namespace sf
+{
+class String;
+}
 
 namespace sf::priv
 {
```

Comment:
Do we prefer this over only opening up `namespace sf` once? There are plenty of files where the only reason we're not using a nested namespace for `sf::priv::` is because we have some forward declarations in the `sf::` namespace.

Reply:
Will make it consistent with existing practice.